DropInDemo/Demo Base/Settings/BraintreeDemoSettings.h

```diff
@@ -17,6 +17,11 @@ typedef NS_ENUM(NSInteger, BraintreeDemoTransactionServiceThreeDSecureRequiredSt
     BraintreeDemoTransactionServiceThreeDSecureRequiredStatusNotRequired = 2,
 };
 
+typedef NS_ENUM(NSInteger, BraintreeDemoTransactionServiceThreeDSecureRequestedVersion) {
```

Comment:
Anyone know if there's a particular reason we say `TransactionService` in these enum names? It makes them soo long, and I'm not sure what additional information it provides.

Reply:
You're definitely right that the naming is weird, but this is probably outside the scope of this PR. We might create a card to fix that up later.